src/sentry/sentry_apps/services/app/model.py

```diff
@@ -153,5 +153,5 @@ class SentryAppInstallationFilterArgs(TypedDict, total=False):
     organization_id: int
     uuids: list[str]
     status: int
-    api_token_id: str
+    api_token_id: int
```

Comment:
This looks good as well, it just gets used for filtering on https://github.com/getsentry/sentry/blob/master/src/sentry/models/integrations/sentry_app_installation.py#L128